nan_implementation_pre_12_inl.h

```diff
@@ -185,7 +186,8 @@ Factory<v8::String>::New(const uint8_t * value, int length) {
   if (wideString.size() == 0) {
     return v8::String::Empty();
   } else {
-    return v8::String::New(&wideString.front(), wideString.size());
+    return v8::String::New(&wideString.front()
+         , static_cast<int>(wideString.size()));
```

Comment:
But I see now that it is only when no length is specified to `widenString`, so yes, should add an assert here.